src/metabase/query_processor/middleware/process_userland_query.clj

```diff
@@ -7,15 +7,13 @@
   event -- see [[metabase.events.view-log]]."
   (:require
    [java-time.api :as t]
+   [metabase.analytics.sdk :as sdk]
    [metabase.events :as events]
    #_
-   [metabase.lib.core :as lib]
+   [metabase.lib.core :as lib] ;; Bryan 7/24: Why are many of these requires commented out??
```

Comment:
Looks like this is from https://github.com/metabase/metabase/pull/45536

I'd support removing the requires - it's easy enough to restore them if the field usage tracking is added back.

Reply:
Or changing the comment to link to that PR/add context. But leaving it in as a question is less ideal

Reply:
I'm just going to remove them, as they'll be in history forever